docs/shone/examples/chemistry.rst

```diff
@@ -16,9 +16,18 @@ Let's construct a atmospheric structure (temperature-pressure curve):
 .. code-block:: python
 
     import numpy as np
+    import matplotlib.pyplot as plt
 
     pressure = np.geomspace(1e-6, 10, 15)  # [bar]
-    temperature = 2300 * (pressure / 0.1) ** 0.1  # [K]
+    temperature = 2300 * (np.geomspace(1e-6, 10, pressure.size) / 0.1) ** 0.1  # [K]
```

Comment:
The way I wrote it was redundant. Here's a suggestion for consistency and clarity:
```suggestion
    temperature = 2300 * (pressure / 0.1) ** 0.1  # [K]
```

Reply:
Bonus points if you make the same correction in your source code on line 38: https://github.com/bmorris3/shone/pull/10/files#diff-01b0298e1f6dacf74ab16c88d2889005257e03744cf3ada15277fcbc254a0aa6R38